src/main/java/mate/jdbc/dao/impl/DriverDaoImpl.java

```diff
@@ -19,19 +20,19 @@
     public Driver create(Driver driver) {
         String query = "INSERT INTO drivers (name, license_number) VALUES (?, ?)";
         try (Connection connection = ConnectionUtil.getConnection();
-                PreparedStatement saveDriverStatement = connection.prepareStatement(query,
+                PreparedStatement statement = connection.prepareStatement(query,
```

Comment:
I think there was no need to change the classes that we already have, return everything back

Reply:
i spoke about it with INikita and Sofiya, and they gave me rights for doing it)